### ISSUE TITLE
Unexpected Error when trying to perform /dim create

### ISSUE THREAD

**Issue by StaticSapphic** [QuiltServerTools/Interdimensional#16]
I'm really sorry for the lack of details with this problem but I've not been given an error message to work with. 

When I try to perform the `/dim create` command I get the error message: `An unexpected error occurred trying to execute that command`

No information is logged to the console, my guess is that it probably is with creating the GUI because I have full permissions and am able to create portals. I am using the following versions of the required mods:
```
Interdimensional 1.1.0
Fantasy 0.4.2+1.17
Polymer 0.2.0-beta.17+1.18.1
Fabric API 0.45.0+1.18
Fabric Language Kotlin 1.7.1+kotlin.1.6.10
```

I created a new server with no mods other than this and the issue still persists

**Comment by ScorchedPsyche** [comment]
I'm also having the same issue at the moment. I'll try to debug this a little bit more and see if I can come up with more information.

There's no error printed at the console.


Using:

- Minecraft 1.18.1
- Interdimensional 1.1.0
- Fabric API 0.45.1
- Language Kotlin 1.7.1 + Kotlin 1.6.10

**Comment by WachumaFox** [comment]
Also having the same issue with no error printed in the console using 1.1.0, and kotlin 1.6.10, fabric api 0.45.1

**Comment by Genau6502** [comment]
This was fixed in 1.1.1

**Comment by Odex64** [comment]
I'm experiencing the same error. Had to build Interdimensional for Minecraft 1.18.2.
